app/main.py

```diff
@@ -1,18 +1,17 @@
 from __future__ import annotations
+from app.players.dwarves.dwarf import Dwarf
+from app.players.elves.elf import Elf
 
 
-def calculate_team_total_rating(players: dict) -> int:
-    total_rate = 0
-    for hero in players:
-        total_rate += hero.get_rating()
-    return total_rate
+def calculate_team_total_rating(players: list) -> int:
+    return sum([hero.get_rating() for hero in players])
```

Comment:
`return sum(hero.get_rating() for hero in players) `To improve the efficiency and readability of the code, you can consider using a generator expression instead of a list comprehension.